src/Mod/PartDesign/PartDesignTests/TestInvoluteGear.py

```diff
@@ -265,7 +265,11 @@ def testUsagePocketInternalGearProfile(self):
         pocket.Reversed = True # need to "pocket upwards" into the cylinder
         pocket.Type = 'ThroughAll'
         self.assertSuccessfulRecompute()
-        self.assertSolid(pocket.Shape)
+        if pocket.Shape.ShapeType == 'Compound':
+            self.assertEqual(len(pocket.Shape.Solids), 1)
+            self.assertSolid(pocket.Shape.Solids[0])
+        else:
+            self.assertSolid(pocket.Shape)
```

Comment:
I don't understand this change here. First, it's unrelated to the issue of the duplicate command definition. And then: Why to we need this at all? The tests asserts that the result of the pocket operation is a solid. In which (valid!) cases it is not a solid, but a compound composed of a single solid?